buddyboss/buddyboss-platform
# Fix Video thumbnail is not getting generated when upload video on iPhone device

### All Submissions:


Fixes #2775 

### How to test the changes in this Pull Request:

1. Open the website on iPhone
2. Upload video
3. See the uploaded video and preview, it should generate thumbnail

### Proof Screenshots or Video
https://ibb.co/gMgmGGP

### Other information:
